lib.rs

```diff
@@ -128,6 +128,7 @@ impl<'a> Drop for Sentinel<'a> {
             self.shared_data.active_count.fetch_sub(1, Ordering::SeqCst);
             if panicking() {
                 self.shared_data.panic_count.fetch_add(1, Ordering::SeqCst);
+                self.shared_data.no_work_notify_all();
```

Comment:
If this line was moved _outside_ this `if panicking()` conditional, could we get rid of [this line](https://github.com/frewsxcv/rust-threadpool/pull/63/files#diff-8c13b5c367b1a0d3dc04da468ea844bdR426)? Would that be the same thing?